src/server/StudioManagementSystem/Controllers/GroupApplicationController.cs

```diff
@@ -33,12 +33,26 @@ public ActionResult<bool> AddGroupInvitation(InvitationDto dto)
 
         return task.Result;
     }
+
+    [HttpPatch()]
```

Comment:
I'd stick with a POST here, it's not obvious from the API that we're directly modifying a group. Nor should it be